clustering/infinispan/extension/src/main/resources/schema/jboss-as-infinispan_10_0.xsd

```diff
@@ -0,0 +1,1499 @@
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="security" type="tns:security" minOccurs="0">
+                <xs:annotation>
+                    <xs:documentation>
+                        Security configuration.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="transaction" type="tns:remote-transaction" minOccurs="0">
+                <xs:annotation>
+                    <xs:documentation>
+                        Transaction configuration.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:sequence>
+        <xs:attribute name="connection-timeout" type="xs:int" default="60000">
+            <xs:annotation>
+                <xs:documentation>
+                    Defines the maximum socket connect timeout before giving up connecting to the server.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="default-remote-cluster" type="xs:string">
+            <xs:annotation>
+                <xs:documentation>
+                    Specifies the default remote server cluster.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="key-size-estimate" type="xs:int" default="64">
+            <xs:annotation>
+                <xs:documentation>
+                    This hint allows sizing of byte buffers when serializing and deserializing keys, to minimize array resizing.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="max-retries" type="xs:int" default="10">
+            <xs:annotation>
+                <xs:documentation>
+                    Sets the maximum number of retries for each request. A valid value should be greater or equals than 0.
+                    Value of 0 means no retry will made in case of a network failure.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="module" type="xs:string" default="org.jboss.as.clustering.infinispan">
+            <xs:annotation>
+                <xs:documentation>Defines the module whose class loader should be used when configuring remote cache container marshaller.</xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="name" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    Uniquely identifies this remote cache container.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="protocol-version" type="tns:protocol-version" default="2.6">
```

Comment:
The default version appears to be now 3.0.

https://github.com/infinispan/infinispan/blob/10.1.7.Final/client/hotrod-client/src/main/java/org/infinispan/client/hotrod/ProtocolVersion.java#L41

Reply:
Ah yes. Good catch.